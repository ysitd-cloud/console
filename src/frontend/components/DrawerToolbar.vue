<template functional>
    <v-toolbar flat class="transparent">
        <v-list class="pa-0">
            <v-list-tile href="https://account.ysitd.cloud/" target="_blank">
                <v-list-tile-avatar>
                    <img :src="props.photo" />
                </v-list-tile-avatar>
                <v-list-tile-content>
                    <v-list-tile-title>{{ props.displayName }}</v-list-tile-title>
                </v-list-tile-content>
            </v-list-tile>
        </v-list>
    </v-toolbar>
</template>
