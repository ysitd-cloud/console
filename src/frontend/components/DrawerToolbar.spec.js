import { shallow } from '@vue/test-utils';
import { createRenderer } from 'vue-server-renderer';

import DrawerToolbar from './DrawerToolbar.vue';

/* globals describe, it, expect */

describe('Component DrawerToolbar', () => {
  it('matches snapshot', () => {
    const renderer = createRenderer();
    const wrapper = shallow(DrawerToolbar, {
      propsData: { photo: 'foo.jpg', displayName: 'bar' },
    });
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
});
