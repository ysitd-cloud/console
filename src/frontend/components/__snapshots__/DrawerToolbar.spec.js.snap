// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component DrawerToolbar matches snapshot 1`] = `
<v-toolbar flat="" class="transparent">
    <v-list class="pa-0">
        <v-list-tile to="[object Object]" target="_blank">
            <v-list-tile-avatar>
                <img>
            </v-list-tile-avatar>
            <v-list-tile-content>
                <v-list-tile-title>bar</v-list-tile-title>
            </v-list-tile-content>
        </v-list-tile>
    </v-list>
</v-toolbar>
`;
